### ISSUE TITLE
Issues with logging in to VM - groups: cannot find name for group ID ###

### ISSUE THREAD

**Issue by MartenvanWezel** [forseti-security/forseti-security#1384]
TL;DR: I believe if your superuser is called 'admin' - we run into issues, perhaps a user/group named 'admin' is already created and causes the problems. Strangely client works better than server.

 - [ ] Which version of Forseti Security you're using (look in `google/cloud/security/__init__.py` in your installation).

RC2

 - [ ] Which module(s) (inventory, scanner, enforcer, explain) you're having trouble with.

VMs. 

 - [ ] Include errors, log output, and host operating system, including installed packages.

On CLIENT:
- Login works, but I do note a message upon login: (see one-before-last line)
```
Connected, host fingerprint: ssh-rsa 2048 A9:A3:A2:FB:8C:87:EC:F6:85:B1:6D:04:7F:B9:14:6C:8E:B6:11:41:45:C1:20:96:B
9:5B:19:55:A7:1A:96:4A
Welcome to Ubuntu 16.04.4 LTS (GNU/Linux 4.13.0-1011-gcp x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  Get cloud support with Ubuntu Advantage Cloud Guest:
    http://www.ubuntu.com/business/services/cloud
6 packages can be updated.
0 updates are security updates.
*** System restart required ***
**groups: cannot find name for group ID 2051164465**
admin_confuseac_at@forseti-client-vm-5952:~$ 
```
Stackdriver:
```
A  Apr  9 16:12:36 forseti-client-vm-5952 systemd[1]: Created slice User Slice of admin_confuseac_at. 
A  Apr  9 16:12:36 forseti-client-vm-5952 systemd[1]: Starting User Manager for UID 2051164465... 
A  Apr  9 16:12:36 forseti-client-vm-5952 systemd[1]: Started Session 368 of user admin_confuseac_at. 
A  Apr  9 16:12:36 forseti-client-vm-5952 systemd[20033]: Reached target Paths. 
```

- On SERVER:
Login does not work. The ssh from GCE console says:
The VM guest environment is outdated and only supports the deprecated 'sshKeys' metadata item. Please follow the steps here to update.

Stackdriver:
```
A  Apr  9 16:13:53 forseti-server-vm-5952 google-accounts: INFO Creating a new user account for admin. 
A  Apr  9 16:13:53 forseti-server-vm-5952 google_accounts_daemon[1537]: useradd: group admin exists - if you want to add this user to that group, use -g. 
A  Apr  9 16:13:53 forseti-server-vm-5952 google-accounts: WARNING Could not create user admin. Command '['useradd', '-m', '-s', '/bin/bash', '-p', '*', 'admin']' returned non-zero exit status 9. 
A  Apr  9 16:17:01 forseti-server-vm-5952 CRON[11023]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly) 
```

- [ ] Note how you deployed (via Deployment Manager, local system, etc.).
Standard git clone, then install from source.

**On server- if I log in with a user NOT named 'admin' everything works.**

**Comment by MartenvanWezel** [comment]
Tried with a 'blank' VM - works fine.
Does forseti use a specific image I need to be aware of? Any unusual settings? How can I replicate the minimum VM setup so we can rule out 'external' factors?

**Comment by mirons-google** [comment]
#1354 seems related

**Comment by blueandgold** [comment]
This seems be a bug in the osLogin role, that can not be fixed in Forseti.  We should file an issue / bug  with the product.

**Comment by MartenvanWezel** [comment]
Note - this is still happening, fresh Forseti install, both on a domain with 'admin@' as the cloud shell user, as well as one without (my user is marten@henry)
